### DISCUSSION TITLE
Deploying for C#(.Net Assembly). 'Solver not found (cutsdp)' cplex bmibnb cant not be founded as well

### DISCUSSION THREAD

**Discussion by ChaoII** [yalmip/YALMIP#1323] [General discussion]
hi,
     Thanks for your open-source.  I need build the matlab code to .net assemble，when I add yalmip and specify the solver such as cutsdp, cplex, bmibnb, bnb , my MATLAB cant work well but in C# program it  throw a except message  such as  solvertime: 0 info: 'Solver not found (cutsdp)' problem: -3 yalmiptime: 0.5050.
thanks.

I will attach my code and error info:
MATLAB:
![image](https://github.com/yalmip/YALMIP/assets/44053467/a14d7ab7-48c1-4c5c-b1aa-d9f5b9f4b482)
![image](https://github.com/yalmip/YALMIP/assets/44053467/f24c1257-a1cb-45ee-ab92-18d7902b6745)
C#:
![image](https://github.com/yalmip/YALMIP/assets/44053467/59518c2f-875b-44e3-adf4-f5148a640ead)

**Comment by johanlofberg** [comment]
https://yalmip.github.io/faq/compilesolver/

**Reply by ChaoII** [reply]
Thank you very much! I'd solved the problem for **cutsdp** solver by the [link](https://yalmip.github.io/faq/compilesolver/) you provided,but I change **cplex** solver, the error is still existing. I hope I can get the solution for more detail (Cplex) from you. thank you

**Reply by johanlofberg** [reply]
You have to include also files that are called and checked for by cutsdp, which would be the interface to cplex, which is something like cplexqcp, cplexlink12100 and call_cplexibm_qcmiqp depending on version (you will have to figure out by looking at definesolvers, and then also debug and  look at which files are called when you run cutsdp)

**Reply by ChaoII** [reply]
Thank you very much. I change the solver(mosek) and build the .Net library successfully ,according to your doc.
